lib/tasks/sample_data/shipping_method_factory.rb

```diff
@@ -47,7 +47,8 @@ def create_delivery(enterprise)
   def create_shipping_method(enterprise, params)
     params[:distributor_ids] = [enterprise.id]
     method = enterprise.shipping_methods.new(params)
-    method.zone = zone
+    method.zones << zone
+    method.shipping_categories << Spree::ShippingCategory.find_or_create_by_name('Default')
```

Comment:
What about abstracting this into a class method that saves us from typing the name every time as we did with StockLocation? we reduce the chances of introducing a bug a great deal as well as having fewer things to think about.

Reply:
I considered doing that but I decided not to do it because this is very different from default stock_location. Default stock location is seed data, it's very important for the system to work.
This DefaultShippingCategory is totally different, this is a value we use to migrate v1 instances with existing data to v2, it's not going to be reused. And here is really just a sample shipping category. Here it could be called "by Air".

Reply:
so, having something like default stock location is tricky to mantain and can induce errors (our life would be easier without it) so by not doing the same here we are not creating another similar fragile situation or source of trouble.